plugins/techdocs-backend/src/plugin.ts

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  DockerContainerRunner,
+  loggerToWinstonLogger,
+  cacheToPluginCacheManager,
+} from '@backstage/backend-common';
+import {
+  coreServices,
+  createBackendPlugin,
+} from '@backstage/backend-plugin-api';
+
+import {
+  Preparers,
+  Generators,
+  Publisher,
+} from '@backstage/plugin-techdocs-node';
+import Docker from 'dockerode';
+import { createRouter } from './service';
+
+/**
+ * The TechDocs plugin is responsible for serving and building documentation for any entity.
+ * @public
```

Comment:
```suggestion
 * @alpha
```

Let's make this an alpha export given that the backend plugin system is still not fully ready and this package is 1.x. It also gives us some opportunity to try this plugin and potentially make breaking changes without publishing a new major version of techdocs.